### PR TITLE
[ci] Add haml-lint

### PR DESCRIPTION
**Pull request by Ana06**
Add haml-lint to the project:
- Add haml-lint gem
- Generate the configuration file with the current offenses (using a modified version of the gem was needed for that: https://github.com/Ana06/haml-lint/tree/fix-todo)
- Add haml_lint rake task
- Add haml-lint to Travis

:bowtie:

### CODE REVIEW

**Comment by Ana06** [comment]
[Add haml-lint](https://trello.com/c/4MLw1sYY/325-add-haml-lint)

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ script:
 dist: trusty
 sudo: required
 env:
+  - TEST_SUITE=haml
```
Comment (hennevogel) [inline .travis.yml]: This is getting out of hand. We should not have a test suite for every linter we do. How about we integrate all of them into on suite?

Comment (Ana06) [inline .travis.yml]: I prefer to have them separately, some of them take a lot of time to run.

Comment (hennevogel) [inline .travis.yml]: Huh? Which one?

Comment (Ana06) [inline .travis.yml]: rspec and the old test suite.

Would you be fine if I added this to the Rubocop test suite? :thinking:

Comment (hennevogel) [inline .travis.yml]: I'm talking about merging only the linter suites, not every suite...

Comment (Ana06) [inline .travis.yml]: ok, I'll do that :wink:

Comment (hennevogel) [inline .travis.yml]: cool

**Pull request merged**